arm-network/2016-09-01/swagger/network.json

```diff
@@ -5704,9 +5704,6 @@
           "description": "path ."
         }
       },
-      "required": [
-        "nextHopType"
```

Comment:
No I see its not used as input type in PS where its considered as required. Its not necessary. Its just used as output type.